src/Symfony/Component/Messenger/Bridge/Redis/Tests/Transport/ListableReceiverFunctionalTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Bridge\Redis\Tests\Transport;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Messenger\Bridge\Redis\Transport\Connection;
+use Symfony\Component\Messenger\Bridge\Redis\Transport\RedisReceiver;
+
+/**
+ * @group integration
+ */
+class ListableReceiverFunctionalTest extends TestCase
+{
+    public function testEndToEndListableReceiverFunctionality()
+    {
+        // Create a mock Redis connection
+        $connection = $this->createMock(Connection::class);
```

Comment:
Mocking the Redis connection does not make much sense to me if the test is an integration test. It should use a real Redis server instead.